Exercises-2-Basic/src/exercises/Num1.java

```diff
@@ -0,0 +1,46 @@
+package exercises;
+
+/* 1.
+Write a Java program to print the sum (addition), multiply, subtract, divide and 
+remainder of two numbers.
+Test Data:
+Input first number: 125
+Input second number: 24
+Expected Output :
+125 + 24 = 149
+125 - 24 = 101
+125 x 24 = 3000
+125 / 24 = 5
+125 mod 24 = 5*/
+
+public class Num1 {
+
+    public static void main(String[] args) {
+
+	int firstnumber = 200;
```

Comment:
instead of writing the two number here, get them from the user.

Reply:
Scanner in = new Scanner(System.in);
   
  System.out.print("Input first number: ");
  int num1 = in.nextInt();
   
  System.out.print("Input second number: ");
  int num2 = in.nextInt();

Reply:
done